data/i18n/en/en.toml

```diff
@@ -234,6 +234,15 @@ other = "(released: "
 [release_date_format]
 other = "2006-01-02"
 
+[release_complete]
+other = "Complete"
+
+[release_schedule]
+other = "Schedule"
+
+[release_changelog]
+other = "Changelog"
+
```

Comment:
Between lines 2 - 3, consider adding:
```
[and]
other = "and"
```

Reply:
I'm OK with this PR being a bit of a compromise. Remember that we can't assume the word order of target localizations.
We're _localizing_, not _translating_.